.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    steps:
+      - checkout:
+          path: ~/ITKPerformanceBenchmarking
+      - run:
+          name: Build Docker testing image
+          command: |
+            docker pull insighttoolkit/performancebenchmarking-test:latest
+            ~/ITKPerformanceBenchmarking/test/Docker/build.sh --cache-from=insighttoolkit/performancebenchmarking-test:latest
```

Comment:
I think you need to specify images from all layer. Adding `debian:9` should do it. This was what I did for the centosbuild images.

The arg would look like this: `--cache-from=debian:9,insighttoolkit/performancebenchmarking-test:latest`


See https://github.com/moby/moby/issues/20316#issuecomment-358260810


Also an interesting article https://circleci.com/blog/how-to-build-a-docker-image-on-circleci-2-0/

Reply:
Nice, thanks for the beta!